swagger_parser/lib/src/utils/utils.dart

```diff
@@ -48,6 +48,25 @@ String ioImport(UniversalComponentClass dataClass) => dataClass.parameters
     ? "import 'dart:io';\n\n"
     : '';
 
+const dartGeneratedFileComment = r'''
```

Comment:
Separate this text to generated file comment and ignore_for_file comment

Reply:
How do you want me to separate them and how do you want me to use them in templates? If using them separately we don't need a config parameter for ignore_for_file?

Reply:
Instead of dartGeneratedFileComment and kotlinGeneratedFileComment, use generated File Comment and ignore file comment (which is used only in dart)